WordPress/Classes/ViewRelated/Site Creation/Wizard/SiteCreator.swift

```diff
@@ -37,22 +37,49 @@ final class SiteCreator {
     /// - Returns: an Encodable object
     ///
     func build() throws -> SiteCreationRequest {
-        guard let domainSuggestion = address else {
+
+        guard let siteName = siteName else {
             throw SiteCreationRequestAssemblyError.invalidDomain
         }
-        let siteName = domainSuggestion.isWordPress ? domainSuggestion.subdomain : domainSuggestion.domainName
-        let siteDesign = design?.slug ?? "default"
+
+        let siteDesign = design?.slug ?? Strings.defaultDesignSlug
 
         let request = SiteCreationRequest(
             segmentIdentifier: segment?.identifier,
             siteDesign: siteDesign,
             verticalIdentifier: vertical?.slug,
-            title: information?.title ?? NSLocalizedString("Site Title", comment: "Site info. Title"),
+            title: information?.title ?? Strings.defaultSiteTitle,
```

Comment:
Just a tiny nit, but `siteDesign` is set similarly and got its own line, maybe make `siteDesign` and `title` match how they're set?